<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBoard</class>
 <widget class="QDialog" name="MainBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>511</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>808</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="monopolyboard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>PLAYER</string>
   </property>
  </widget>
  <widget class="QLabel" name="board_pname">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>CURRENTLY AT</string>
   </property>
  </widget>
  <widget class="QLabel" name="board_at">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>166</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>MONEY</string>
   </property>
  </widget>
  <widget class="QLabel" name="board_wallet">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="board_roll">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>250</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>ROLL</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>320</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="board_mycards">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>My Cards</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Mortgage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="board_trade">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Trading Options</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="board_end">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Declare Bankruptcy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
